sds_data_manager/lambda_code/SDSCode/query_api.py

```diff
@@ -37,15 +37,15 @@ def lambda_handler(event, context):
     query_params = event["queryStringParameters"]
 
     # select the file catalog for the query
```

Comment:
```suggestion
    # select the table for the query
```

Reply:
Made a lot of text updates like this with a global find and replace, thanks for finding this!